makelove/macos.py

```diff
@@ -0,0 +1,245 @@
+import io
+import os
+import shutil
+import struct
+import sys
+from datetime import datetime
+from zipfile import ZipFile
+from urllib.request import urlopen, urlretrieve, URLError
+
+from PIL import Image
+
+from .util import eprint, tmpfile
+from .windows import get_default_love_binary_dir, get_download_url
```

Comment:
I think you should put these two functions into `util.py`. I'd prefer if the files for the different targets didn't require each other.